[Feature] React 18 Support Needed

# **Feature Request**

## **Is your feature request related to a problem? Please describe.**

React 18 Dependency Error

## **Describe the solution you'd like**

I'm using this package in one of my projects and it's super awesome. But I have to force install this package because of dep conflict with React 18.  It would be great if you could add support for React 18.

## **Describe alternatives you've considered**

N/A

## **Additional context**

npm ERR! While resolving: myproject@1.0.0
npm ERR! Found: react@18.1.0
npm ERR! node_modules/react
npm ERR!   react@"18.1.0" from the root project
npm ERR!   peer react@"^17.0.2 || ^18.0.0-0" from next@12.1.6
npm ERR!   node_modules/next
npm ERR!     next@"12.1.6" from the root project
npm ERR!     peer next@"^12.0.0 || ^13.0.0" from next-cloudinary@0.8.0
npm ERR!     node_modules/next-cloudinary
npm ERR!       next-cloudinary@"*" from the root project
npm ERR!   1 more (react-dom)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^18.2.0" from next-cloudinary@0.8.0
npm ERR! node_modules/next-cloudinary
npm ERR!   next-cloudinary@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.

--- a/next-cloudinary/package.json
+++ b/next-cloudinary/package.json
@@ -26,7 +26,7 @@
     "microbundle": "^0.15.1"
   },
   "peerDependencies": {
-    "next": "^12.0.0 || ^13.0.0",
-    "react": "^18.2.0"
+    "next": "^12 || ^13",
+    "react": "^17 || ^18"
   }
 }

